normandy/recipes/api/v3/serializers.py

```diff
@@ -153,18 +153,10 @@ def validate_extra_filter_expression(self, value):
     def validate(self, data):
         data = super().validate(data)
         action = data.get("action")
-        required_error_msg = serializers.PrimaryKeyRelatedField.default_error_messages["required"]
-
         if action is None:
-            if not self.instance:
-                raise serializers.ValidationError({"action_id": required_error_msg})
```

Comment:
The old validation here strikes me as really odd, so I tracked down when it was added. It was in commit 49dd7adbdeb997f46bf3c6c7764d15b4de2bf783 / PR #1240, though I don't really understand why. I think we haven enough test coverage in this area though, so I'm just going to chalk it up to code is weird sometimes. @rehandalal you wrote that PR. Do you know why this is the way it is?

Reply:
my brain appears to have dropped all info on why i did whatever i did

Reply:
Well, in that case I'll just chalk it up to a mystery. Lets go ahead and land this.